vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/conversion_frequency_cap.h

```diff
@@ -22,6 +22,8 @@ class ConversionFrequencyCap final : public ExclusionRule<CreativeAdInfo> {
   ConversionFrequencyCap(const ConversionFrequencyCap&) = delete;
   ConversionFrequencyCap& operator=(const ConversionFrequencyCap&) = delete;
 
+  std::string GetUuid(const CreativeAdInfo& creative_ad) const override;
```

Comment:
GetUUid should return the id for the frequency cap, i.e. creative instance id, creative set id, campaign id or advertiser id. This value should match the value logged in GetLastMessage for each frequency cap